src/utils/openUrl.ts

```diff
@@ -3,11 +3,9 @@
  *  Licensed under the MIT License. See License.md in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import * as open from 'open';
+import { env, Uri } from 'vscode';
 
 export async function openUrl(url: string): Promise<void> {
     // Using this functionality is blocked by https://github.com/Microsoft/vscode/issues/85930
-    // await vscode.env.openExternal(vscode.Uri.parse(url));
-
-    await open(url);
+    await env.openExternal(Uri.parse(url));
```

Comment:
This is still broken apparently. We may have to special-case the logic to use open for desktop, but I think all of our scenarios work with `env.openExternal`

Reply:
Related: https://github.com/microsoft/vscode-azuretools/pull/1401#discussion_r1136195032